spark/CMakeLists.txt

```diff
@@ -0,0 +1,110 @@
+################################################################################
```

Comment:
does it make sense to name the folder spark-integration?

Reply:
@xwang2713 does it make sense to rename the folder 'spark' to 'spark-integration'?